src/routes/(site)/blog/[id]/blog-body.svelte

```diff
@@ -1,5 +1,41 @@
 <script lang="ts">
+  import { onMount } from 'svelte';
   export let data = '';
+  onMount(() => {
+    let pre = document.querySelectorAll('pre');
+    let svg = `<svg xmlns="http://www.w3.org/2000/svg" class="ionicon" viewBox="0 0 512 512"
+  ><rect
+    x="128"
+    y="128"
+    width="336"
+    height="336"
+    rx="57"
+    ry="57"
+    fill="none"
+    stroke="currentColor"
+    stroke-linejoin="round"
+    stroke-width="32"
+  /><path
+    d="M383.5 128l.5-24a56.16 56.16 0 00-56-56H112a64.19 64.19 0 00-64 64v216a56.16 56.16 0 0056 56h24"
+    fill="none"
+    stroke="currentColor"
+    stroke-linecap="round"
+    stroke-linejoin="round"
+    stroke-width="32"
+  /></svg
+>`;
```

Comment:
It might be possible to import the raw string for this SVG via `import CopyTextSVG from '$lib/components/copy-text.svelte?inline"`.

Reply:
```
failed to load module for ssr: /src/lib/components/copy-text.svelte?inline
Error: failed to load module for ssr: /src/lib/components/copy-text.svelte?inline
    at instantiateModule (file:///Users/danieltruong/Desktop/acmcsuf.com/node_modules/vite/dist/node/chunks/dep-665b0112.js:53253:15)
 ```

Reply:
how come svgs in svelte files and not svg file tht can be put in img tags?

Reply:
> ```
> 
> failed to load module for ssr: /src/lib/components/copy-text.svelte?inline
> 
> Error: failed to load module for ssr: /src/lib/components/copy-text.svelte?inline
> 
>     at instantiateModule (file:///Users/danieltruong/Desktop/acmcsuf.com/node_modules/vite/dist/node/chunks/dep-665b0112.js:53253:15)
> 
>  ```

Followed up in #623.